.env.example

```
@@ -1,40 +1,32 @@
+# Web API Key (Required for web interface authentication)
+# Generate a secure key using:
+#   node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
+# Or use any random 64-character hex string
+WEB_API_KEY=your-secret-api-key-here
 
-# OPTIONAL: Web Interface Port
-# Port for the web management interface
-# Default: 3000
-WEB_PORT=3000
+# Web Interface Port (Optional, default: 3000)
+WEB_PORT=3000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)